src/main/java/patio/voting/domain/VotingStats.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (C) 2019 Kaleidos Open Source SL
+ *
+ * This file is part of Don't Worry Be Happy (DWBH).
+ * DWBH is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * DWBH is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DWBH.  If not, see <https://www.gnu.org/licenses/>
+ */
+package patio.voting.domain;
+
+import io.micronaut.data.annotation.DateCreated;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import patio.common.domain.utils.Builder;
+
+/**
+ * Represents all the statistics which belong to a {@link Voting}
+ *
+ * @since 0.1.0
+ */
+@Entity
+@Table(name = "voting_stats")
+public final class VotingStats {
+
+  @Id @GeneratedValue private UUID id;
+
+  @OneToOne
+  @JoinColumn(name = "voting_id", referencedColumnName = "id")
+  private Voting voting;
+
+  @DateCreated
+  @Column(name = "created_at")
+  private OffsetDateTime createdAtDateTime;
+
+  private Float average;
```

Comment:
Why average is `Float` and movingAverage is `Double` ? Both should be of the same type

Reply:
It seems comprehensible. I choosed Float becouse of the precision type returned from the db query.

But either one should be move to float, or the other to long. I agree